tests/phpunit/tests/media.php

```diff
@@ -3872,8 +3923,13 @@ public function test_wp_filter_content_tags_does_not_lazy_load_first_image_in_bl
 	 *
 	 * @ticket 56930
 	 *
+	 * @since 6.3.0 Apply fetchpriority="high" on the LCP image.
```

Comment:
```suggestion
	 * @ticket 58235
	 *
	 * @since 6.3.0 Apply fetchpriority="high" on the LCP image.
```
Added the ticket number that made this change.